en/docs/setup/configuring-a-jdbc-user-store.md

```diff
@@ -217,6 +217,20 @@ true : Write groups to user store<br />
 false : Do not write groups to user store, so only internal roles can be created. Depending on the value of ReadGroups property, it will read existing groups from user store or not</p></td>
 </tr>
 <tr class="odd">
+<td>PasswordHashMethod</td>
+<td>password_hash_method</td>
+<td>Password Hashing Algorithm</td>
+<td><p>Specifies the Password Hashing Algorithm used the hash the password before storing in the user store.<br />
+Possible values:<br />
+SHA - Uses SHA digest method. SHA-1, SHA-256<br />
+MD5 - Uses MD 5 digest method.<br />
+PLAIN_TEXT - Plain text passwords.</p>
+<p>If you just configure as SHA, It is considered as SHA-1, It is always better to configure algorithm with higher bit value as digest bit size would be increased.
```

Comment:
Shall we break this down? Please look out for the capitalizations as well. Avoid using capital letters after commas.